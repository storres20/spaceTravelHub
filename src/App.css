body {
  padding: 0 20px;
}

.active {
  text-decoration: underline;
}