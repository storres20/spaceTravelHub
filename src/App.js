import React from 'react';
import './App.css';
import {
  Routes, Route,
} from 'react-router-dom';
import NavbarSpace from './components/NavbarSpace';
import Rockets from './pages/Rockets';
import Missions from './pages/Missions';
import MyProfile from './pages/MyProfile';

const App = () => (
  <>
    <NavbarSpace />

    <Routes>
      <Route path="/" element={<Rockets />} />
      <Route path="/missions" element={<Missions />} />
      <Route path="/myProfile" element={<MyProfile />} />
    </Routes>
  </>
);

export default App;
