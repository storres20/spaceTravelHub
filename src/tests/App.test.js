import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
// Redux
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
// ****
import App from '../App';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <BrowserRouter>
          <App />
        </BrowserRouter>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
