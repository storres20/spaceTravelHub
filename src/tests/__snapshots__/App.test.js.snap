// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <nav
    className="navbar navbar-expand-md navbar-light"
  >
    <div
      className="container-fluid"
    >
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="planet"
          className="NavImg"
          src="planet.png"
        />
      </a>
      <h2
        className="NavTitle"
      >
        Space Travelerâ€˜s Hub
      </h2>
      <button
        aria-controls="navbarScroll"
        aria-label="Toggle navigation"
        className="navbar-toggler collapsed"
        onClick={[Function]}
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        aria-expanded={null}
        className="justify-content-end navbar-collapse collapse"
        id="navbarScroll"
      >
        <div
          className="my-2 my-lg-0 NavFlex navbar-nav navbar-nav-scroll"
          onKeyDown={[Function]}
          style={
            Object {
              "maxHeight": "100px",
            }
          }
        >
          <a
            aria-current="page"
            className="menu active"
            href="/"
            onClick={[Function]}
            style={
              Object {
                "textDecoration": "underline",
              }
            }
          >
            Rockets
          </a>
          <a
            className="menu"
            href="/missions"
            onClick={[Function]}
          >
            Missions
          </a>
          <span>
            |
          </span>
          <a
            className="menu"
            href="/myProfile"
            onClick={[Function]}
          >
            My Profile
          </a>
        </div>
      </div>
    </div>
  </nav>,
  <div
    className="d-grid card-group"
    style={
      Object {
        "borderStyle": "none",
        "marginTop": "2%",
        "padding": "10px",
      }
    }
  />,
]
`;
